SpatialGDK/Source/SpatialGDK/Private/Utils/RPCContainer.cpp

```diff
@@ -187,20 +194,19 @@ void FRPCContainer::BindProcessingFunction(const FProcessRPCDelegate& Function)
 	ProcessingFunction = Function;
 }
 
-bool FRPCContainer::ApplyFunction(FPendingRPCParams& Params)
+ERPCQueueCommand FRPCContainer::ApplyFunction(FPendingRPCParams& Params)
 {
 	ensure(ProcessingFunction.IsBound());
 	FRPCErrorInfo ErrorInfo = ProcessingFunction.Execute(Params);
 
 	if (ErrorInfo.Success())
 	{
-		return true;
+		return ERPCQueueCommand::ContinueProcessing;
 	}
-	else
-	{
+
 #if !UE_BUILD_SHIPPING
-		LogRPCError(ErrorInfo, QueueType, Params);
+	LogRPCError(ErrorInfo, QueueType, Params);
 #endif
-		return ErrorInfo.bShouldDrop;
-	}
+
+	return ErrorInfo.QueueCommand;
```

Comment:
We only seem to set this to `StopProcessing` in `SendLegacyRPC` (which I left a comment about), but previously, any RPC that had a result other than `Success` would get queued, so perhaps the default value for `QueueCommand` should be `StopProcessing`?

Reply:
In fact, `QueueCommand` is only used when the result is not success, so perhaps we could reflect that in the name? e.g. `QueueCommandOnFailure`